tools/config/__init__.py

```diff
@@ -696,7 +734,8 @@ def _assign_new_offset(rom_start, start, new_offset, region_name):
                 "Can not place % region inside previous region" % region_name)
         return newstart
 
-    def _generate_bootloader_build(self, rom_start, rom_size):
+    def _generate_bootloader_build(self, **active_rom_regions):
```

Comment:
How about we drop `**`, as it adds no value, and we do regions -> memories, and we drop the word "active", as it's not adding anything. 
```suggestion
    def _generate_bootloader_build(self, rom_memories):
```